docs/Doxyfile

```diff
@@ -743,7 +743,7 @@ WARN_LOGFILE           =
 # spaces.
 # Note: If this tag is empty the current directory is searched.
 
-INPUT                  = docs/mainpage.dox AdsLib/AdsDef.h AdsLib/AdsLib.h
+INPUT                  = docs/mainpage.dox AdsLib/
```

Comment:
Hmm, I like the general idea of this pull request, but is the result of any value? The most documented file is AdsLib/AdsLib.h and I find no functions from it in the output in your fork.
How do you use that output yourself?